renovate.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["config:base", ":gitSignOff"],
+  "schedule": ["before 3pm every Weekday"]
```

Comment:
Not a lot of deps, so I don't expect this to be noisy.  The main goal is to catch pomerium updates automatically.